Jenkinsfile

```diff
@@ -65,21 +66,50 @@ pipeline {
                             }
                         }
                         stage("Docker-py") {
+                            steps {
+                                ansiColor('xterm') {
```

Comment:
I noticed the output was _sometimes_ wrong, and _sometimes_ looked good; I initially disabled color output with `--color=no` on `pytest`, but we actually didn't need a TTY for these tests, so removing the `--t` did the right thing (https://github.com/moby/moby/pull/39716)

Reply:
i've removed the `ansiColor` commit